### PR TITLE
fix: bank query build error

### PR DESCRIPTION
**Pull request by KimuraYu45z**
we can use this if we set 

```
denoms?: string[]
```

as `undefined`.

### CODE REVIEW

**Comment by KimuraYu45z** [comment]
```
git cherry-pick 1a6411d
```

**Comment by Senna46** [comment]
Why use `declare const QueryApi: any | { balance(address: string, denom: string): Promise<any> };`?
Can't all queries use `cosmosclient.rest.bank`

**Comment by KimuraYu45z** [comment]
This is a temporal fixing.
Currently all pages use with `denoms = undefined` so currently there is no problem.
After the update of `cosmos-client`, we will not have to use `QueryApi` directly so this code is completely temporal stuff.

**Comment by KimuraYu45z** [comment]
We now just use `allBalance` that is exported by `cosmos-client` without `QueryApi`.
Now we don't use `balance` that is not exported by `cosmos-client` and inevitable to use `QueryApi` directly.

**Comment by Senna46** [comment]
Same problem as https://github.com/UnUniFi/web-apps/pull/327 ?
It's better to match.

File: projects/portal/src/app/models/cosmos/bank.query.service.ts [hunk 1]
```diff
@@ -20,18 +22,21 @@ export class BankQueryService {
...
     return this.restSdk$.pipe(
       mergeMap((sdk) =>
         Promise.all(
-          denoms.map((denom) =>
-            new QueryApi(undefined, sdk.url)
-              .balance(address, denom)
-              .then((res) => res.data.balance!),
+          denoms.map(
+            (denom) =>
+              new QueryApi(undefined, sdk.url)
```
Comment (Senna46) [inline projects/portal/src/app/models/cosmos/bank.query.service.ts:37]: I do not reccomend use QueryApi bacause of  using `any` type.

**Review by Senna46 — CHANGES_REQUESTED** [review]

**Comment by KimuraYu45z** [comment]
https://github.com/UnUniFi/web-apps/pull/327/commits/dae1fbc809558d4738fabec843bf354e4d533165

**Pull request closed without merging**